app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -15,4 +15,32 @@
     color: $secondary-text-colour;
     text-decoration: none;
   }
+
+  @include media-down(mobile) {
+    &.collapse-on-mobile li {
+      display: none;
+
+      &.parent-taxon {
+        background-image: none;
+        display: block;
+        margin-left: 0;
+        padding-left: 14px;
+        position: relative;
+
+        &:before {
+          border-bottom: 5px solid transparent;
```

Comment:
I tried adding govuk-elements to the project, but it's not straightforward to include the arrow in the sass because govuk-elements provides the whole back link, including arrow, which doesn't match our styles here. There are also conflicts between the `@media` breakpoints, since we just want this to appear on mobile but the govuk-elements elements applies at all levels.

It might be nice to extract the arrow as a mixin and use it in both places, but I suggest doing it later once we've decided we're going to stick with this pattern. It's also going to take some thought (and discussion with the frontend teams) about how to add it to the library as a  reusable component, because it doesn't quite fit what they have already.